docs: updated clean command page per issue description (#1431)
kurtosis-tech/kurtosis: docs/docs/cli-reference/clean.md

slug: /clean
---

> The `clean` command is used to remove enclaves and related processes, often to free up resources. By default this command affects stopped enclaves. If the -a flag is set, all running enclaves and engine containers will be removed.  

Reviewer: ```suggestion
The `clean` command is used to remove stopped enclaves and their contents to free up resources. If the `-a` flag is passed in, all running enclaves, their contents, and unused engine containers will be removed as well.
```

- Removes stopped enclaves and stopped engine containers

> - Removes all services within enclaves

Reviewer: ```suggestion
- Removes all services within stopped enclaves
```

- Removes all unused Kurtosis images (engine + logs aggregator + enclaves)

> - Removes all files artifacts and all docker volumes within those enclaves

Reviewer: ```suggestion
- Removes all files artifacts and all Docker volumes within stopped enclaves
```
